friends/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import FriendList from './components/FriendList';
+import Form from './components/Form';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+    }
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends')
+      .then(response => {
+        console.log(response);
+        this.setState({ friends: response.data })
+      })
+      .catch(error => {
+        console.log(error);
```

Comment:
Why have an extra step here in your `catch`'s callback? You can just use `console.log` since `console` is an object and `log` a method on it that'll take however many arguments you pass it and log them:
```javascript
.catch(console.log); /* Error object is passed to console.log implicitly, 
just like an Event object being passed implicitly in event handlers for onClicks */
```